lib/TPP/Transforms/Utils/BuilderUtils.cpp

```diff
@@ -35,13 +35,15 @@ arith::ConstantOp getConstant(OpBuilder &builder, Type type, ValueT value) {
 } // anonymous namespace
 
 func::FuncOp createFunction(OpBuilder &builder, ModuleOp module, StringRef name,
-                            TypeRange args, TypeRange ret) {
+                            TypeRange args, TypeRange ret, bool createBody) {
```

Comment:
IIRC, your removed code had private visibility, while this function gives public visibility. It should not matter for our purposes, just pointing out because you manually set it. Making sure you're not missing anything.

Reply:
I didnt want to change the API.I can either add an argument, similarly as the createBody flag, or just set the function as private after creating the prototype. [Here's why](https://mlir.llvm.org/getting_started/Faq/#whats-with-op-symbol-declaration-cannot-have-public-visibility). What do you recommend?